cube/src/main/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversion.java

```diff
@@ -177,19 +198,42 @@ public ModelToJsonConversion(Configuration configuration, ApplicationProfile app
     this(configuration, new ApplicationProfileToConversionProfile().apply(applicationProfile));
   }
 
+  public ConversionReport applyAndReport(Model model, String root) {
+    Context context = createContext(model);
+    ObjectNode conversionResult = apply(context, root);
+    return createConversionReport(context, conversionResult);
+  }
+
+  private ConversionReport createConversionReport(Context context, ObjectNode conversionResult) {
+    ConversionReport result = new ConversionReport();
+    result.setResult(conversionResult);
+
+    Model missedTriples = JenaUtils.difference(context.model, context.alreadyProcessedModel);
```

Comment:
note: it might be we have less issues because we are using JenaUtils.difference. Maybe we should test this.